cypress/integration/tests/administrator/proxy/proxy.enable.disable.test.ts

```diff
@@ -22,6 +22,27 @@ describe("Proxy operations", () => {
         preservecookies();
     });
 
+    it("Http Proxy validation", function () {
+        proxy.httpEnabled = true;
+        proxy.enableInvalidProxy();
+        proxy.disable();
+    });
```

Comment:
So create some locators in proxy.view like httpPort httpHost and methods like fillhttpPort() and fillhttpHost() etc and then import them and fill values in test and check if validation message is shown and submit button is disabled.



    it("Http Proxy validation", function () {
        proxy.enableSwitch(`#${type}Proxy`);
        fillhttpHost(hostname);
        fillhttpPort( "Invalid port");
        cy.get("#port-helper").contains("This field is required");
        cy.get(submitButton).should("be.disabled");

Reply:
Example : https://github.com/konveyor/tackle-ui-tests/blob/d6cc7a9fcd056e7a8fa2873c4b4ab856bdd587f7/cypress/integration/tests/developer/applicationinventory/assessment/copy_assessment/copy_assessment.test.ts#L161

Reply:
Solved in a9c764a5336d595c30dd97b8ee17ad01872b0ef1.